benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -659,8 +679,12 @@ void GraphicsBenchmarkApp::RecordCommandBuffer(PerFrame& frame, grfx::SwapchainP
 
     // Record commands for the scene using one renderpass
     frame.cmd->BeginRenderPass(currentRenderPass);
-    RecordCommandBufferSkybox(frame);
-    RecordCommandBufferSpheres(frame);
+    if (pEnableSkyBox->GetValue()) {
```

Comment:
we need to disable the `currentRenderPass` when both skybox and spheres are disabled?

Reply:
Done.

Reply:
sorry i couldn't find this change to disable the render pass when both both skybox and spheres are disabled, is it in a different PR?

Reply:
Sorry, when I rebased this PR with the last commits, I realized the Enable/Disable knobs for Skybox and Spheres were not working as it used to be and it seems the issue is related to recent commits. But now they were fixed with https://github.com/google/bigwheels/pull/331, so I will wait to this to be merged to update.

Reply:
Updated!